app/javascript/src/components/Dashboard/Notes/Card/Card.jsx

```diff
@@ -0,0 +1,43 @@
+// @ts-nocheck
+import React from "react";
+
+import { MenuVertical, Clock } from "@bigbinary/neeto-icons";
```

Comment:
It seems to me that you haven't gone through the Slack thread very well. Go through it, fix the whole PR and then add a comment and re-request my review once done.

Reply:
I have gone through the slack thread and identified not using the aliases in my code.
I have corrected the known mistakes and I am sending another PR along with the Notes deletion UI.
Is that fine?